Server/prisma/seed.ts

```diff
@@ -0,0 +1,61 @@
+import { PrismaClient } from "@prisma/client"
+import dotenv from "dotenv"
+import CryptoService from "../src/services/crypto.service"
+import database from "../src/services/database.service"
+import { logger } from "../src/utils/logger"
+
+const prisma = new PrismaClient()
+
+/**
+ * This script initialises the database with first entries.
+ * (See docs: https://www.prisma.io/docs/guides/migrate/seed-database)
+ *
+ * The database schema is generated via `prisma db push`.
+ * This script only initialises the database data if there is no data!
+ */
+async function main() {
+	// Import variables from .env into process.env
+	dotenv.config()
+	const config = process.env
+
+	// If no user exists: Create Initial User
+	// Intial Users credentials are stored in environment variables
+	// for security reasons
+	if ((await database.users.getAll()).length == 0) {
+		const pwd = await CryptoService.produceHash(config.INIT_PWD!)
+		await database.users.save({ username: config.INIT_USR!, password: pwd! })
+	}
+
+	// If no type for POIs exist: Create Initial POITypes
+	if ((await database.pois.getAllTypes()).length == 0) {
+		// POIType: Level Crossing
+		await database.pois.saveType({ name: "Level Crossing", icon: "level_crossing.svg" })
```

Comment:
The POI Type Icons are now strings containing integers. The mapping between these is found in the Enum `POITypeIcon` in `src/models/api.ts`. So for example `"1"`

Reply:
Ah, Thanks ! I will change it accordingly